app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,7 @@
 
 <nav id="header" class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
     <!-- ||| menu, displayed for mobile: -->
-  <a class="navbar-brand ml-lg-5" id="brand" href="<%= root_path %>">Public Lab</a>
+  <a class="navbar-brand ml-lg-5" id="brand" href=" /home">Public Lab</a>
```

Comment:
Hi @CoderJolly, nice work! But instead of `"/home"` I'd like to suggest you used `<%= home_path %>` instead. This way whenever the routes are changed it'll automatically reflect here, otherwise, the link will break... :)

Reply:
And, actually, doesn't this need to stay as `root_path` here because it should be different depending on if you're logged in or not? Just checking, i could be wrong! If logged in, it should go to `/home` but if not, to `/` -- make sense?